app/templates/ui/app/error/errorInterceptor.service.js

```diff
@@ -25,6 +25,8 @@
             };
           }
           messageBoardService.message(msg);
+          var toastMsg = '<strong>' + msg.title + '</strong><p>' + msg.body + '</p>';
+          toast.danger(toastMsg);
```

Comment:
Sounds a bit like overkill, showing a message on the messageBoard as well as via toasts? But maybe I'd have to see first.. I think you can test by adding a constraint for an index that doesn't exist. Search will then break, and should normally trigger the messageboard..